projects/common-form-elements/src/lib/dynamic-form/dynamic-form.component.ts

```diff
@@ -360,4 +360,11 @@ export class DynamicFormComponent implements OnInit, OnChanges, OnDestroy  {
     return null;
     // return result ? {compare: true} : null;
   }
+  checkSectionLevelCss(section) {
```

Comment:
why is this function required? Why not just check the expression directly in the template? 
`section?.renderingHints?.class || 'grid two-column-grid'`

Reply:
yes.. @santoshilimi  dont need this.

Reply:
Done